pkg/handler/api.go

```diff
@@ -10,7 +10,7 @@ import (
 	"path/filepath"
 	"strconv"
 
-	"github.com/content-services/content-sources-backend/docs"
+	spec_api "github.com/content-services/content-sources-backend/api"
```

Comment:
Would love input on this naming @jlsherrill

Reply:
that looks fine to me!